From ba7fa583bf7fdc548e961f23d2801cb4e7d50aba Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 23 Jan 2012 16:08:46 +0100
Subject: [PATCH] 99base/init: remove tmpfs on /dev

tmpfs on /dev is not supported anymore and devtmpfs is supported in
kernels for a long time.
---
 modules.d/99base/init |   11 +----------
 1 files changed, 1 insertions(+), 10 deletions(-)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 33a7379..abfa3c5 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -110,16 +110,7 @@ if [ "$RD_DEBUG" = "yes" ]; then
 fi
 
 if ! ismounted /dev; then
-    # try to mount devtmpfs
-    if ! mount -t devtmpfs -o mode=0755,nosuid devtmpfs /dev >/dev/null 2>&1; then
-        # if it failed fall back to normal tmpfs
-        mount -t tmpfs -o mode=0755,nosuid tmpfs /dev >/dev/null 2>&1
-        # Make some basic devices first, let udev handle the rest
-        mknod -m 0666 /dev/null c 1 3
-        mknod -m 0666 /dev/ptmx c 5 2
-        mknod -m 0600 /dev/console c 5 1
-        mknod -m 0660 /dev/kmsg c 1 11
-    fi
+    mount -t devtmpfs -o mode=0755,nosuid devtmpfs /dev >/dev/null 2>&1
 fi
 
 # prepare the /dev directory
